app/src/main/java/pl/example/android/garbageapp/data/database/SectorTermDao.java

```diff
@@ -12,8 +12,8 @@
 @Dao
 public interface SectorTermDao {
 
-    @Query("SELECT * FROM sector_terms WHERE term >= :term AND sectorType == :sectorType")
-    LiveData<List<SectorTerm>> getFutureSectorTerms(Date term, int sectorType);
+    @Query("SELECT * FROM sector_terms WHERE term >= :t AND sectorType == :sectorType")
+    LiveData<List<SectorTerm>> getCurrentSectorTerms(Date t, SectorType sectorType);
```

Comment:
Change param name, 't' is not correct.

Reply:
You are right. I was checking which one "term" or ":term" refers to method arguments and I left it accidentally.